source/openvino.js

```diff
@@ -126,7 +126,16 @@ openvino.Graph = class {
         this._arguments = {};
 
         for (const layer of this._const(net.layers, net.edges)) {
-            const inputs = layer.inputs.map((input) => this._argument(layer.id, layer.precision, input, net.edges));
+            const inputs = layer.inputs.map((input) => {
+                const to = layer.id + ':' + input.id;
+                const [outputLayerId, outputId] = net.edges[to].split(':');
+                const outputLayer = net.layers.find((layer) => layer.id === outputLayerId);
```

Comment:
Lookup map and `find` can be inefficient.

Reply:
What lookup map can you suggest?

Reply:
`const layers = new Map(net.layers.map((entry) => [ entry.id, entry ]))` and `layers.get(outputLayerId))`.